Resources/DesktopModules/Admin/Dnn.PersonaBar/Modules/Dnn.Pages/App_LocalResources/pages.nl-NL.resx

```diff
@@ -91,25 +91,25 @@
     <value xml:space="preserve">Bulk pagina instellingen</value>
   </data>
   <data name="BulkPagesLabel.Text" lastModified="2016-11-29 19:18:02">
-    <value xml:space="preserve">Een pagina per regel, voorafgaand met "&amp;gt;"om hierarchie te creeeren.</value>
+    <value xml:space="preserve">Een pagina per regel, voorafgaand met "&gt;"om hiërarchie te creeëren.</value>
```

Comment:
Een pagina per regel, voorafgaand met "&gt;"om hiërarchie te creëren.

Reply:
@aspelt Ik denk dat het handig is dat de huidige updates worden doorgevoerd en dat we daarna de verbeteringen aanbrengen

Reply:
@EPTamminga Is wat mij betreft prima. Fijn dat we zo de vertaling met elkaar kunnen bijwerken. Goed bezig!!!